.gitignore

```diff
@@ -1,3 +1,8 @@
+.DS_Store
+*.iml
+.classpath
+.project
+.idea/
```

Comment:
I once also tried to add those IntelliJ Files to the Project's gitignore - without success. So maybe you should move the to you system's ~/.gitignore file.